projects/ui-components/lf-metadata/lf-field-template-container/lf-field-template-container.component.ts

```diff
@@ -378,11 +462,17 @@ export class LfFieldTemplateContainerComponent extends LfFieldContainerDirective
           }
           return validFieldType;
         });
-      } catch (err: any) {
+        this.templateState = TemplateState.SHOW_TEMPLATE;
+      }
+      catch (err: any) {
         this.templateErrorMessage = this.TEMPLATE_HAS_FAILED_TO_LOAD;
-        console.error('getTemplateFieldsAsync failed: ' + err.message);
+        console.error('getTemplateFieldsAsync failed: ' + err?.message ?? err?.title ?? '');
+        this.templateState = TemplateState.HAS_ERROR;
```

Comment:
why are we throwing the error here? 
I think since we handle it in the UI, we shouldn't throw again